dev-requirements.txt

```diff
@@ -10,4 +11,7 @@ loguru==0.4.1
 tensorflow==2.2.0
 ruamel.yaml==0.16.10
 pytest==5.4.3
-flake8==3.8.3
\ No newline at end of file
+flake8==3.8.3
+twine==3.2.0
+setuptools==50.3.0
+wheel==0.35.1
```

Comment:
Isn't this related to another [PR](https://github.com/SubstraFoundation/distributed-learning-contributivity/pull/246/files)?

Reply:
Yes it is 
I think I cherrypicked it but I cannot remember why